### ISSUE TITLE
Support for GitHub Merge Queues

### ISSUE THREAD

**Issue by JakeWhitakerSempre** [AikidoSec/github-actions-workflow#48]
Currently Aikido fails to run if triggered on a Merge queue event.

```
Error: start scan failed: {"status_code":400,"reason_phrase":"Missing field: base_commit_id"}
```

The GitHub action trigger is different for a merge queue so the payload does not match.

Github docs:

https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-a-merge-queue

https://docs.github.com/en/webhooks/webhook-events-and-payloads#merge_group

**Comment by Kemosabert** [comment]
Hey Jake, 
Thanks for flagging this. We've created a patch for this in PR #50 and it will be available in version 1.0.12 of the action. I will let you know once its released.

**Comment by Kemosabert** [comment]
@JakeWhitakerSempre version v1.0.12 has been released and is ready for usage in merge queues 👍
